src/main/java/com/meilisearch/sdk/Search.java

```diff
@@ -40,6 +40,11 @@ String rawSearch(String uid, String q) throws Exception {
      * @param attributesToRetrieve Attributes to display in the returned documents
      * @param attributesToCrop Attributes whose values have been cropped
      * @param cropLength Length used to crop field values
+     * @param cropMarker String to customize default crop marker, default value: …
```

Comment:
I think this could be improved to something more informative, without a context the user will not know what is a `crop marker` :)

Reply:
I modify it, it is better?